GVFS/GVFS.Hooks/Program.cs

```diff
@@ -85,8 +85,8 @@ private static void RunPreCommands(string[] args)
         {
             if (ProductUpgrader.IsLocalUpgradeAvailable())
             {
-                Console.WriteLine("A newer version of GVFS is available.");
-                Console.WriteLine("Run \"gvfs upgrade\" on \"Command Prompt\" as Administrator to install.");
+                Console.Out.WriteLine("A newer version of GVFS is available.");
+                Console.Out.WriteLine("Run \"gvfs upgrade\" on \"Command Prompt\" as Administrator to install.");

```

Comment:
Why did you switch to `Console.Out.WriteLine`?

Reply:
Only to make it explicit that we are writing to stdout. (Git hooks stdout is getting redirected to stderr. During testing the nag messages were showing in stderr.)